fdbserver/CommitProxyServer.actor.cpp

```diff
@@ -1331,6 +1351,17 @@ ACTOR Future<Void> assignMutationsToStorageServers(CommitBatchContext* self) {
 					MutationRef backupMutation(
 					    MutationRef::Type::ClearRange, intersectionRange.begin, intersectionRange.end);
 
+					// TODO (Nim): Encrypt properly once logic for clear range encryption is finalized
```

Comment:
am bit confused with this comment, what does "encrypt properly" refer to? Are we referring to identifying "encryption domain"? If yes, then we can explicitly write code as 

if ((MutationType::Type)m.type == MutationType::ClearRange) {
...
} else {
...
}

Reply:
This comment is referring to the fact that currently a clear range is encrypted using the system encryption key. Once you add the logic to handle clear ranges (which can cross tenant boundaries) I can implement something similar here